src/code/middleware/state-save.js

```diff
@@ -0,0 +1,29 @@
+import actionTypes from '../action-types';
+import { getClassId, getUserId } from '../utilities/url-params';
+
+export const authoringVersionNumber = 1;
```

Comment:
I played around with the idea of dynamically figuring out which version of the authoring document is getting used, and incrementing the version number if it doesn't exist in Firebase. I couldn't figure out a way to do this without querying the root of the database and getting back all the data.

I suppose if we had a separate mapping of version numbers to authoring documents somewhere else in Firebase, we could do this more easily.